common/env/testdata/subset_env.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "subset-env"
+stdlib:
+  exclude_macros:
+    - map
+    - filter
+  exclude_functions:
```

Comment:
In Java, we treated configuring both include/exclude in the same path to be an error: https://github.com/google/cel-java/blob/main/checker/src/test/java/dev/cel/checker/CelStandardDeclarationsTest.java#L58

It's likely useful to do the same here to avoid surprises

Reply:
I don't have a great builder pattern I can follow with how the YAML serialization works. What I can do is add validation that only one of these options is set when trying to configure an environment from a config object or attempt to subset and provide both options, something in the `StdLibSubset()` option that I can invoke on the `LibrarySubset` object to validate it's correctness. What do you think?

Reply:
SGTM, I think it'd be good to inform the user somehow pertaining to misconfiguration. I'll defer to you on where that's done.